app/(gcforms)/[locale]/(form administration)/form-builder/[id]/publish/Publish.tsx

```diff
@@ -161,12 +161,16 @@ export const Publish = ({ id }: { id: string }) => {
     }
 
     try {
-      updateTemplate({
+      const operationResult = await updateTemplate({
         id,
         name: getName(),
         formConfig,
       });
 
+      if (operationResult === null) {
+        throw new Error("Failed to publish form");
+      }
+
       router.push(`/${i18n.language}/unlock-publishing`);
```

Comment:
@timarney  Not sure how this piece of code worked before. Maybe it never really worked? Since it was in a try/catch block I assume it was expected that the server action would throw something but it was not the case.

Reply:
Ya your right this must have been missed along the way